kibbeh/public/locales/bn/translation.json

```diff
@@ -216,7 +216,7 @@
       "editProfileModal": {
         "usernameTaken": "ইউজার নাম নেওয়া হয়ে গিয়েছে",
         "avatarUrlError": "অবৈধ ছবি",
-        "avatarUrlLabel": "GitHub/Twitter এর avatar url",
+        "avatarUrlLabel": "GitHub/Twitter এর Avatar URL",
```

Comment:
Why did you make the `a` capital there?

Reply:
If there's a English word after a Bengali word, we usually make the English word's first letter uppercase

Reply:
Got it